### ISSUE TITLE
Trial design bug

### ISSUE THREAD

**Issue by ch728** [solgenomics/sgn#3471]
Using the newest docker image (v22) I see the following errors when designing trials:

![weird2](https://user-images.githubusercontent.com/36375878/114601157-2f0cde80-9c63-11eb-8fec-e1d0b75ec240.png)

![weird1](https://user-images.githubusercontent.com/36375878/114600967-f66d0500-9c62-11eb-9be3-0e43b01d08eb.png)

Notice it's starting on plot number 11 and doing some wonkiness with inserting blank plots.

**Comment by chris263** [comment]
It should be due to agricolae (R package) which is used to randomize and get the plot number. In agricolae has 3 options to get the plot number (1 - start numbering with 11. It is ideal when having less than 10 plots per block) (2 - start with 101) (3 - start with 1001).
In this particular case of this ticket, I guess the ideal is having the plot number with 101.

The trial design page is showing start with 1, the thing to do maybe is change it to 11.

![Screen Shot 2021-04-15 at 11 55 49 AM](https://user-images.githubusercontent.com/51970212/114900325-c304b480-9de1-11eb-8736-36f3c6836a22.png)

**Comment by ch728** [comment]
yeah, that's a problem, but the bigger problem is the first picture I show. The default for no layout shouldn't look like that... bunch of blank plots and the trial was suppose to have 70 plots but it only has 55 shown. The second example is a totally  different example which is when I specify the number of rows with 10 accessions and three reps. Looks more normal, but the plot number is off. I think plot one should start with.... plot one if that is what is chosen, not plot 11 and there shouldn't be some blank plots inserted in the first example. Kind of confusing for people in the field wouldn't you agree?

**Comment by ch728** [comment]
I would have to think most breeding programs would want to have a regular numbering 1...n and not this 101...10000000000000000001 thing. People need to reason about the actual coordinates and plot numbering in the field easily or chaos will ensue.

**Comment by chris263** [comment]
Yes, the first image is really weird. We have to take a look to find out what is happening. Maybe @aco46 could comment on that.
Regarding the plot numbering, breeders actually ask to start numbering plots with 101, it makes some space to get the second block starting with 201 and the third with 301, and so on. It is very common having that rather continuously numbering.

**Comment by ch728** [comment]
Yeah, I see. Everyone has different preferences I suppose. I see the logic in the other numbering system, but would there be a way to preserve/allow continuous numbering as existed in earlier BreedBase versions? A lot of our pipelines are reliant on continuous numbers. It's what our technicians are used to and I always hesitate to change stuff like this on them.

**Comment by chris263** [comment]
Sure, do you think they need continuous numbering for all designs or for a specific one?

**Comment by ch728** [comment]
I can't imagine us using a design outside of RCBD, Augmented or unreplicated (nursery)  in the near future, but it might be nice to have that feature across all designs for the sake of consistent options for the user.  I am probably way oversimplifying it, I haven't worked with agricolae, but would it be just a matter of sorting by the plot number agricolae assigns and then data$plot_sequential<- 1:nrow(data) if continuous option is chosen?
